src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -640,6 +644,25 @@ where symbol.Equals(symbol.ContainingType.FindImplementationForInterfaceMember(m
             }
         }
 
+        private void GenerateExtensionMethods(INamedTypeSymbol symbol, MetadataItem item)
+        {
+            var extensions = new List<string>();
+            foreach (var e in _extensionMethods)
+            {
+                var reduced = e.ReduceExtensionMethod(symbol);
+                if ((object)reduced != null)
+                {
+                    // update reference
+                    var typeGenericParameters = reduced.ContainingType.IsGenericType ? reduced.ContainingType.Accept(TypeGenericParameterNameVisitor.Instance) : EmptyListOfString;
```

Comment:
the class that defines extension methods must be non-generic class, this way i think we could leave out the parameter typeGenericParameters?